src/common/Constants.cpp

```diff
@@ -55,7 +55,7 @@ ReadAccountsFromConstantsFile(std::string propName)
         auto child = acc.second.get_optional<std::string>(propName);
         if (child)
         {
-            // LOG_MESSAGE("constants " << child.get());
+            // LOG_GENERAL("constants " << child.get());
```

Comment:
Remove redundant code directly

Reply:
Not responsible for this part of code, used GLOBAL replacement for LOG_MESSAGE->LOG_GENERAL

Reply:
I think it is ok to remove. If the author of the code need it back, he can easily add it since it is currently not in used.